ush/python/pygfs/task/snow_analysis.py

```diff
@@ -357,13 +357,20 @@ def finalize(self) -> None:
         FileHandler(yaml_copy).sync()
 
         logger.info("Copy analysis to COM")
-        template = f'{to_fv3time(self.task_config.current_cycle)}.sfc_data.tile{{tilenum}}.nc'
+        bkgtimes = []
+        if self.task_config.DOIAU:
+            # need both beginning and middle of window
+            bkgtimes.append(self.task_config.SNOW_WINDOW_BEGIN)
+        bkgtimes.append(self.task_config.current_cycle)
         anllist = []
-        for itile in range(1, self.task_config.ntiles + 1):
-            filename = template.format(tilenum=itile)
-            src = os.path.join(self.task_config.DATA, 'anl', filename)
-            dest = os.path.join(self.task_config.COM_SNOW_ANALYSIS, filename)
-            anllist.append([src, dest])
+        for bkgtime in bkgtimes:
+            template = f'{to_fv3time(bkgtime)}.sfc_data.tile{{tilenum}}.nc'
+            for itile in range(1, self.task_config.ntiles + 1):
+                filename = template.format(tilenum=itile)
```

Comment:
```suggestion
```
This looks like a duplicate line to the next.

Reply:
Good catch, lol